server/battleships/domain/src/test/java/eu/optas/domain/CounterTest.java

```diff
@@ -0,0 +1,20 @@
+package eu.optas.domain;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.*;
+
+class CounterTest {
+    @Test
+    void name() {
+//        LocalDate mock = mock(LocalDate.class);
+//        when(mock.toString()).thenReturn("EXPECTED");
```

Comment:
^

Reply:
Šitą palieku kaip pavyzdį ateičiai.

Reply:
Kode tokių šiukšlių nepaliekam. Jei sau kaip pavyzdį naudosi, tai užsirašyk kažkur į notepad ar scratch'ą

Reply:
šito nereikėjo comittint